cve_bin_tool/checkers/ntp.py

```diff
@@ -16,6 +16,7 @@ class NtpChecker(Checker):
     FILENAME_PATTERNS = [r"ntp"]
     VERSION_PATTERNS = [
         r"ntpd ([0-9]+\.[0-9]+\.[0-9]+p[0-9]+)",
+        r"ntpdate ([0-9]+\.[0-9]+\.[0-9]+p[0-9]+)",
         r"ntpdc ([0-9]+\.[0-9]+\.[0-9]+p[0-9]+)",
         r"ntpq ([0-9]+\.[0-9]+\.[0-9]+p[0-9]+)",
```

Comment:
I was wondering if we could reduce the number of iterations by condensing them into a single regex pattern like `ntp(?:d|date|dc|q) ([0-9]+\.[0-9]+\.[0-9]+p[0-9]+)`.

We could do it on another PR since the refactor is out of scope of this one.

Reply:
Feel free to file a separate issue for the refactoring if think about it!